iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -2112,4 +2119,11 @@ public List<FileStatus> getMergeTaskInputFiles(Properties properties) throws IOE
   public MergeTaskProperties getMergeTaskProperties(Properties properties) {
     return new IcebergMergeTaskProperties(properties);
   }
+
+  public void setCustomDeleteProperties(TableDesc tableDesc) {
+    tableDesc.getProperties().put(InputFormatConfig.OPERATION_TYPE_PREFIX + tableDesc.getTableName(),
```

Comment:
do we need both `OPERATION_TYPE_PREFIX` and `WRITE_OPERATION_CONFIG_PREFIX`.
what are they used for?

Reply:
`OPERATION_TYPE_PREFIX` is used while configuring job conf.
Whereas `WRITE_OPERATION_CONFIG_PREFIX` is used during delete record writer creation.